app/build.gradle

```diff
@@ -78,12 +78,12 @@ dependencies {
     kapt "com.google.dagger:dagger-compiler:$daggerVersion"
     compile "com.google.dagger:dagger:$daggerVersion"
 
-    androidTestCompile "org.mockito:mockito-core:1.10.19"
+    testCompile "junit:junit:4.12"
+    testCompile "org.mockito:mockito-core:$mockitoVersion"
+
+    androidTestCompile "org.mockito:mockito-core:$mockitoVersion"
     androidTestCompile "com.google.dexmaker:dexmaker:1.2"
```

Comment:
I think that you could also extract `1.2` to `dexmakerVersion` value as it is used twice.